lib/search/bloc/search_bloc.dart

```diff
@@ -46,6 +52,14 @@ class SearchBloc extends Bloc<SearchEvent, SearchState> {
       _getTrendingCommunitiesEvent,
       transformer: throttleDroppable(throttleDuration),
     );
+    on<VoteCommentEvent>(
```

Comment:
Eventually, we might want to combine these general events into one single event (similar to how FeedBloc and InstanceBloc handles it) to make it a bit cleaner to work with!

Reply:
Absolutely agreed, there's a lot of duplication around comments. Might make sense to have a `CommentBloc` at some point.